jobs/pingcap/tidb-engine-ext/latest/ghpr_unit_test.groovy

```diff
@@ -0,0 +1,84 @@
+// REF: https://<your-jenkins-server>/plugin/job-dsl/api-viewer/index.html
+// For trunk and latest release branches.
+pipelineJob('pingcap/tidb-engine-ext/ghpr_unit_test') {
+    logRotator {
+        daysToKeep(90)
```

Comment:
30 or 90?

Reply:
Low trigger frequency, retaining three months is more reasonable